infrastructure-playbooks/rolling_update.yml

```diff
@@ -697,6 +697,7 @@
 
   hosts:
     - "{{ iscsi_gw_group_name|default('iscsigws') }}"
```

Comment:
Why is this different than all the other instances ?

Reply:
What do you mean?

Reply:
Oh I see, well that's a different way to write it.